src/blocks/separator/Separator.tsx

```diff
@@ -0,0 +1,47 @@
+import { forwardRef } from 'react';
+import styled from 'styled-components';
+
+import { SeparatorCSSProps, SeparatorComponentProps } from './Separator.types';
+import { getSeparatorResponsiveCSS, separatorCSSPropsKeys } from './Separator.utils';
+import { BlockWithoutStyleProp } from 'blocks/Blocks.types';
+import { getBlocksColor } from 'blocks/Blocks.utils';
+
+export type SeparatorProps = SeparatorCSSProps & SeparatorComponentProps & BlockWithoutStyleProp<HTMLDivElement>;
+
+const StyledSeparator = styled.div.withConfig({
+  shouldForwardProp: (prop, defaultValidatorFn) =>
+    !separatorCSSPropsKeys.includes(prop as keyof SeparatorCSSProps) && defaultValidatorFn(prop),
+})<SeparatorProps>`
+  /* Initial values */
+  width: ${({ width, orientation }) => width || (orientation === 'horizontal' ? 'auto' : '1px')};
+  height: ${({ height, orientation }) => height || (orientation === 'horizontal' ? '1px' : 'auto')};
```

Comment:
When the orientation is horizontal, the separator should automatically take the full width of the parent container.

When the orientation is vertical, the separator should automatically take the full height of the parent container.

until a width or height is passed as a prop from outside.

Right now I have to pass width/height:100% everytime if I wan't to render a full length container.